packages/core/src/mesh/ModelMesh.ts

```diff
@@ -1205,11 +1333,37 @@ export class ModelMesh extends Mesh {
     this._uv5 = null;
     this._uv6 = null;
     this._uv7 = null;
+    this._indices = null;
```

Comment:
Missed `_boneIndices` and `_boneWeights`?

Reply:
done